test/integration/scheduler/preemption_test.go

```diff
@@ -932,98 +933,209 @@ func TestPreemptionRaces(t *testing.T) {
 	}
 }
 
-// TestNominatedNodeCleanUp checks that when there are nominated pods on a
-// node and a higher priority pod is nominated to run on the node, the nominated
-// node name of the lower priority pods is cleared.
-// Test scenario:
-// 1. Create a few low priority pods with long grade period that fill up a node.
-// 2. Create a medium priority pod that preempt some of those pods.
-// 3. Check that nominated node name of the medium priority pod is set.
-// 4. Create a high priority pod that preempts some pods on that node.
-// 5. Check that nominated node name of the high priority pod is set and nominated
-//    node name of the medium priority pod is cleared.
-func TestNominatedNodeCleanUp(t *testing.T) {
-	// Initialize scheduler.
-	testCtx := initTest(t, "preemption")
-	defer testutils.CleanupTest(t, testCtx)
+const (
+	alwaysFailPlugin = "alwaysFailPlugin"
+	doNotFailMe      = "do-not-fail-me"
+)
 
-	cs := testCtx.ClientSet
+type alwaysFail struct{}
```

Comment:
Add a comment for the functionality of this plugin

Reply:
done.